charts/prometheus/templates/server/deploy.yaml

```diff
@@ -94,13 +104,12 @@ spec:
           {{- range .Values.server.extraFlags }}
             - --{{ . }}
           {{- end }}
-          {{- if .Values.server.baseURL }}
-            - --web.external-url={{ .Values.server.baseURL }}
-          {{- end }}
-
```

Comment:
Please no change if no functional gain.

Reply:
As stated in PR description, its just an alignment with STS that I can push as new PR if you prefer.

Reply:
New PR please.

Reply:
Change pushed to a new PR: https://github.com/prometheus-community/helm-charts/pull/1125